components/automate-ui/src/app/pages/team-details/team-details.component.spec.ts

```diff
@@ -254,5 +312,14 @@ describe('TeamDetailsComponent', () => {
       });
     });
   });
-});
 
+  function genProjectOption(
+    label: string, checked?: boolean, value?: string): ProjectsFilterOption {
+    return <ProjectsFilterOption>{
```

Comment:
❓ Do we need the cast? Doesn't TS check that we return something that's valid, as declared in line 317?